examples/chef/nrfconnect/Kconfig

```diff
@@ -15,6 +15,7 @@
 #
 mainmenu "Matter nRF Connect Chef Example Application"
 
-rsource "../../../config/nrfconnect/chip-module/Kconfig.defaults"
-rsource "../../../config/nrfconnect/chip-module/Kconfig.features"
+#rsource "../../../config/nrfconnect/chip-module/Kconfig.defaults"
```

Comment:
Why this change? you don't need to source `chip-module/Kconfig` explicitly as it will be loaded automatically by `Kconfig.zephyr`.